components/mpas-ocean/src/shared/mpas_ocn_vmix.F

```diff
@@ -865,7 +869,7 @@ subroutine ocn_vel_vmix_tend_implicit_spatially_variable_mannings(forcingPool, b
          if (N .eq. Nsurf) then
            normalVelocity(N,iEdge) = normalVelocity(N,iEdge)  &
                 / (1.0_RKIND + dt*implicitCD &
```

Comment:
Should we consider whether the thickness in the manning's coeff.'s above (via `implicitCd`) should also be the harmonic one? Perhaps this is a bridge too far, and would require some wrangling to work in both the single- and multi-layer cases.

Reply:
Good question. Hmm, let me look into whether I can easily test this with the drying slope test case.

Reply:
@dengwirda The legacy test case for variable mannings is broken sometime at/before v2.1. I'm going to see whether there's a quick fix that would enable us to maintain this capability. In the meantime, I'm pushing the changes I am testing in accordance with your suggestions.

Reply:
I put the harmonic treatment of layer thicknesses (here, water column thickness) into the `implicitCd` computation. I was able to test out these changes in the legacy test case for mannings drag https://github.com/E3SM-Ocean-Discussion/E3SM/pull/36/commits/c7751f0d75a33a6e1092e40c0da7e78a95f7ea71.